db/schema.rb

```diff
@@ -67,6 +67,7 @@
   create_table "messages", force: :cascade do |t|
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string   "emailadd"
```

Comment:
This shouldn't be here. Can you update your branch with the latest changes from `integration` and run `db:migrate` again? It should update your `schema.rb`